introduction.md

```diff
@@ -0,0 +1 @@
+My name is Jeff Dembinski and I am practicing forking, cloning, and other git related topics. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.